# MPC parameters
predict_vars:
  x: 0.5
  z: 0.5
  dx: 500
  dz: 500
  att: 0.05
  datt: 10000
  thrust_scaling: 1
  disturbance_torque: 50